src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2595,6 +2595,28 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
         "usgovvirginia": "usgovvirginia"
     }
 
+    # mapping for JEDI - USNat
+    AzureUSNatLocationToOmsRegionCodeMap = {
+        "usnateast": "USNATEAST",
+        "usnatwest": "USNATWEST"
+    }
+
+    AzureUSNatRegionToOmsRegionMap = {
+        "usnateast": "usnateast",
+        "usnatwest": "usnatwest"
+    }
+
+    # mapping for JEDI - USSec
+    AzureUSSecLocationToOmsRegionCodeMap = {
+        "usseceast": "USSECEAST",
+        "ussecwest": "USSECWEST"
+    }
+
+    AzureUSSecRegionToOmsRegionMap = {
+        "usseceast": "usseceast",
+        "ussecwest": "ussecwest"
+    }
```

Comment:
Is it allowed to expose the settings of JEDI cloud in code? Please double confirm. 

You may consider add `LocationToOmsRegionCodeMap` and `RegionToOmsRegionMap` as endpoints of a cloud. [cloud.py](https://github.com/Azure/azure-cli/blob/f5ecf9e2aca18c3409689bda5e8d892b97942e15/src/azure-cli-core/azure/cli/core/cloud.py#L56).

Reply:
@fengzhou-msft - This still puts it in the code.
Is there any way i can get this information from another place without having to put it in the open source code?
Are you aware of any other teams which are doing this?

Reply:
Spoke to Fengzhou offline. Right now the API support is stil being built and I have updated the code to work around this problem.